apps/docs/src/lib/config.ts

```diff
@@ -55,6 +79,17 @@ function getConfiguration(
   );
 }
 
+const getDeployEnvironment = () => {
```

Comment:
would favour function definitions (`function ...`) over function expressions (`const .. = `)

Reply:
could also type the return as `DocumentSiteEnvironment`